eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -7,11 +7,15 @@
 
     <!-- Baseline 7.0 dependencies until msbuild targets net8 and uses a net8 arcade, SBRP, etc.
          These cannot be added to 7.0 SBRP, because they would are produced in the 7.0 build. -->
```

Comment:
does the comment still apply?

Reply:
No, I don't think so. I'll follow up in a later PR (since this is delaying VS work I want to get it in).